src/util/atomic_io.c

```diff
@@ -87,7 +87,9 @@ ssize_t sss_atomic_read_safe_s(int fd, void *buf, size_t buf_len, size_t *_len)
     }
 
     if (ulen > buf_len) {
-        return ERANGE;
+        *_len = ulen;
```

Comment:
Check below suggests `_len` can be `NULL`.
Even if this is not the case in the real code base, we surely will get a covscan failure here.

Reply:
Thanks, I added a check for this.

Reply:
Thank you.